src/instruments/src/PFD/AttitudeIndicatorHorizon.jsx

```diff
@@ -0,0 +1,192 @@
+                <path id="FlightPathDirector" d="m69.914 80.823a1.0073 1.0079 0 1 0-2.0147 0 1.0073 1.0079 0 1 0 2.0147 0zm7.5551 0 6.5478-1.5119v3.0238l-6.5478-1.5119m-17.125 0-6.5478-1.5119v3.0238l6.5478-1.5119h17.125" className="NormalStroke Green" />
+            </g>
+        </g>
+    );
+};
+
+const RadioAltAndDH = ({ radioAlt, decisionHeight, roll }) => {
+    if (radioAlt <= 2500) {
+        const verticalOffset = calculateVerticalOffsetFromRoll(roll);
+        const size = (radioAlt > 400 ? 'FontLarge' : 'FontLargest');
+        const DHValid = decisionHeight >= 0;
+        const color = (radioAlt > 400 || (radioAlt > decisionHeight + 100 && DHValid) ? 'Green' : 'Amber');
+
+        let text = '';
+
+        if (radioAlt < 5) {
+            text = Math.round(radioAlt).toString();
+        } else if (radioAlt <= 50) {
+            text = (Math.round(radioAlt / 5) * 5).toString();
+        } else if (radioAlt > 50 || (radioAlt > decisionHeight + 100 && DHValid)) {
+            text = (Math.round(radioAlt / 10) * 10).toString();
+        }
+
+        return (
+            <g id="DHAndRAGroup" transform={`translate(0 ${-verticalOffset})`}>
+                {radioAlt <= decisionHeight ? <text id="AttDHText" x="73.511879" y="113.19068" className="FontLargest Amber EndAlign Blink9Seconds">DH</text> : null}
+                <text id="RadioAlt" x="68.803764" y="119.88165" className={`${size} ${color} MiddleAlign`}>{text}</text>
+            </g>
+        );
+    }
+    return null;
+};
+
+const SideslipIndicator = ({ isOnGround, roll }) => {
+    let SIIndexOffset = 0;
+
+    const verticalOffset = calculateVerticalOffsetFromRoll(roll);
+
+    if (isOnGround) {
+        // on ground, lateral g is indicated. max 0.3g, max deflection is 15mm
+        const latAcc = getSimVar('ACCELERATION BODY X', 'G Force');
+        const accInG = Math.min(0.3, Math.max(-0.3, latAcc));
+        SIIndexOffset = -accInG * 15 / 0.3;
+    } else {
+        SIIndexOffset = Math.max(Math.min(getSimVar('INCIDENCE BETA', 'degrees'), 15), -15);
+    }
+
+    return (
+        <g id="RollTriangleGroup" transform={`translate(0 ${verticalOffset})`} className="NormalStroke Yellow CornerRound">
+            <path d="m66.185 43.176h5.4437l-2.7218-3.427z" />
+            <path id="SideSlipIndicator" transform={`translate(${SIIndexOffset} 0)`} d="m62.862 47.359 2.1615-2.7214h7.7653l2.1615 2.7214z" />
+        </g>
+    );
+};
+
+const RisingGround = ({ radioAlt, pitch }) => {
+    const targetPitch = -0.1 * radioAlt;
+
+    const targetOffset = Math.max(Math.min(calculateHorizonOffsetFromPitch(pitch - targetPitch) - 31.563, 0), -63.093);
+
+    return [
+        <path d="m113.95 157.74h-90.08v-45.357h90.08z" transform={`translate(0 ${targetOffset})`} className="NormalOutline EarthFill" />,
+        <path id="HorizonGroundRectangle" d="m113.95 157.74h-90.08v-45.357h90.08z" transform={`translate(0 ${targetOffset})`} className="NormalStroke White" />,
+    ];
```

Comment:
I'd suggest using React fragments if possible over arrays, and only use arrays when you're truly working with a dynamic list where the length can change:

```suggestion
    return <>
        <path d="m113.95 157.74h-90.08v-45.357h90.08z" transform={`translate(0 ${targetOffset})`} className="NormalOutline EarthFill" />,
        <path id="HorizonGroundRectangle" d="m113.95 157.74h-90.08v-45.357h90.08z" transform={`translate(0 ${targetOffset})`} className="NormalStroke White" />,
    </>;
```

Reply:
Ah, didn't know this was possible, I'll change this as soon as I can. Thanks!